module/GeebyDeeby/view/partials/searchbox.phtml

```diff
@@ -3,7 +3,7 @@
     Search:
     <input type="text" name="SearchQuery" value="<?=isset($this->layout()->query) ? $this->escapeHtml($this->layout()->query) : ''?>" size="35" maxlength="50" />
     <select name="SearchType">
-      <? foreach (array('Title', 'Person', 'Keyword', 'ISBN') as $current): ?>
+      <? foreach (['Title', 'Person', 'Keyword', 'ISBN'] as $current): ?>
```

Comment:
Change to short array syntax is something else that really belongs in master, not this PR.